module/zfs/dmu_send.c

```diff
@@ -837,17 +992,23 @@ dmu_send_impl(void *tag, dsl_pool_t *dp, dsl_dataset_t *to_ds,
 	}
 #endif
 
-	if (large_block_ok && to_ds->ds_feature_inuse[SPA_FEATURE_LARGE_BLOCKS])
+	/* raw sends imply large_block_ok */
+	if ((large_block_ok || rawok) &&
```

Comment:
Encrypted datasets can't have embedded data since I need to store the MAC somewhere, even for authenticated objects.